pkg/kubelet/kubelet_pods.go

```diff
@@ -1632,13 +1632,15 @@ func (kl *Kubelet) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kubecontaine
 		pod.Spec.Containers,
 		len(pod.Spec.InitContainers) > 0,
 		false,
+		false,
 	)
 	apiPodStatus.InitContainerStatuses = kl.convertToAPIContainerStatuses(
 		pod, podStatus,
 		oldPodStatus.InitContainerStatuses,
 		pod.Spec.InitContainers,
 		len(pod.Spec.InitContainers) > 0,
 		true,
+		true,
```

Comment:
Since we want to allow startup probes for sidecars, which are init containers, I'd suggest we do this change somehow differently. Maybe set it to false here, but than change to `true` around the time when the prober manager would be started for them.

Reply:
But does it make sense for ephemeral containers to transition Started false->true in the prober manager - since they have no probes?
It kinda makes sense for init containers since we'll add them soon...